### PR TITLE
Hotkeys to launch the 3 main tools

### PR DESCRIPTION
**Pull request by dabreegster**
Also fix the "focused" style, so it's more clear what mode is active, even when you use a hotkey

### CODE REVIEW

File: src/lib/draw/Toolbox.svelte [hunk 1]
```diff
@@ -44,15 +59,15 @@ repeatedly load anything. Make sure this is only created once, then just hidden.
   <div class="toolbar">
     <ToolButton setMode={{ mode: "new-point" }}>
       <img src={pointIcon} alt="New point" />
-      New point
+      New point (1)
```
Comment (Pete-Y-CS) [inline src/lib/draw/Toolbox.svelte:62]: idk how I feel about. I don't think the average user will interpret this as 'this number represents a hotkey'

**Review by Pete-Y-CS — COMMENTED** [review]
Change is good, but I don't know how we communicate it to users

File: src/lib/draw/Toolbox.svelte (hunk 1 above)
Comment (dabreegster) [reply src/lib/draw/Toolbox.svelte:62]: I tried underlining (used in some dropdown menus to indicate hotkeys), but it looked odd. We could also hide the info in the help for each tool, but that's _really_ undiscoverable. Hmm..

Comment (dabreegster) [reply src/lib/draw/Toolbox.svelte]: PTAL -- stopped showing the hotkey upfront, moved to help

**Pull request merged**